persistence-modules/spring-jpa-2/src/main/java/com/baeldung/multitenant/MultiTenantApplication.java

```diff
@@ -0,0 +1,14 @@
+package com.baeldung.multitenant;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+
+@SpringBootApplication(exclude = { DataSourceAutoConfiguration.class })
+public class MultiTenantApplication {
+
+	public static void main(String[] args) {
```

Comment:
Formatting: this should not be indented this far.

Reply:
Done